### PR TITLE
Remove bearer-only occurences in the documentation when possible. Mak…

### LINKED ISSUE #23066: Remove bearer-only content from the Server Administration Guide
### Description

This feature is now implemented differently.  There is not bearer-only option, but it appears many places including this chapter: https://www.keycloak.org/docs/latest/server_admin/index.html#assembly-managing-clients_server_administration_guide

### Discussion

_No response_

### Motivation

_No response_

### Details

_No response_

### PR DESCRIPTION
**Pull request by mposolda**
…e the documentation clearer for client-access-type client policy

closes #23066

### CODE REVIEW

File: docs/documentation/server_admin/topics/clients/client-policies.adoc [hunk 1]
```diff
@@ -69,7 +69,7 @@ Author of a client (Checked by presence to the particular role or group)::
     like the administrator of the {project_name}.
 
 Client Access Type (confidential, public, bearer-only)::
-    For example when a client sends an authorization request, a policy is adopted if this client is confidential.
+    For example when a client sends an authorization request, a policy is adopted if this client is confidential. Confidential client has enabled client authentication when public client has disabled client authentication. Bearer-only is deprecated client type.
```
Comment (andymunro) [inline docs/documentation/server_admin/topics/clients/client-policies.adoc:72]:
```suggestion
    For example when a client sends an authorization request, a policy is adopted if this client is confidential. Confidential client has enabled client authentication when public client has disabled client authentication. Bearer-only is a deprecated client type.
```

File: docs/documentation/server_admin/topics/clients/oidc/con-audience.adoc [hunk 2]
```diff
@@ -56,12 +56,13 @@ When setting up audience checking:
 [[_audience_resolve]]
 == Automatically add audience
 
-An _Audience Resolve_ protocol mapper is defined in the default client scope _roles_. The mapper checks for clients that have at least one client role available for the current token. The client ID of each client is then added as an audience, which is useful if your service (usually bearer-only) clients rely on client roles.
+An _Audience Resolve_ protocol mapper is defined in the default client scope _roles_. The mapper checks for clients that have at least one client role available for the current token. The client ID of each client is then added as an audience, which is useful
+if your service clients rely on client roles. Service client could be usually a client without any flows enabled, which may not have any tokens issued directly to itself. It represents OAuth 2 _Resource Server_.
```
Comment (andymunro) [inline docs/documentation/server_admin/topics/clients/oidc/con-audience.adoc:60]:
```suggestion
if your service clients rely on client roles. Service client could be usually a client without any flows enabled, which may not have any tokens issued directly to itself. It represents an OAuth 2 _Resource Server_.
```

File: js/apps/admin-ui/public/locales/en/translation.json [hunk 3]
```diff
@@ -2284,7 +2284,7 @@
   "defaultLocaleHelp": "The initial locale to use. It is used on the login screen and other screens in the Admin UI and Account UI.",
   "conditionsHelp": "Conditions, which will be evaluated to determine if client policy should be applied during particular action or not.",
   "clientProfilesHelp": "Client profiles applied on this policy.",
-  "clientAccessType": "It uses the client's access type (confidential, public, bearer-only) to determine whether the policy is applied. Condition is checked during most of OpenID Connect requests (Authorization requests, token requests, introspection endpoint request, etc.)",
+  "clientAccessType": "It uses the client's access type (confidential, public, bearer-only) to determine whether the policy is applied. Condition is checked during most of OpenID Connect requests (Authorization requests, token requests, introspection endpoint request, etc.). Confidential client has enabled client authentication when public client has disabled client authentication. Bearer-only is deprecated client type.",
```
Comment (andymunro) [inline js/apps/admin-ui/public/locales/en/translation.json:2287]:
```suggestion
  "clientAccessType": "It uses the client's access type (confidential, public, bearer-only) to determine whether the policy is applied. Condition is checked during most of OpenID Connect requests (Authorization requests, token requests, introspection endpoint request, etc.). Confidential client has enabled client authentication when public client has disabled client authentication. Bearer-only is a deprecated client type.",
```

File: js/apps/admin-ui/public/locales/en/translation.json [hunk 4]
```diff
@@ -3153,7 +3153,7 @@
   },
   "client-accesstype": {
     "label": "Client Access Type",
-    "tooltip": "Access Type of the client, for which the condition will be applied."
+    "tooltip": "Access Type of the client, for which the condition will be applied. Confidential client has enabled client authentication when public client has disabled client authentication. Bearer-only is deprecated client type."
```
Comment (andymunro) [inline js/apps/admin-ui/public/locales/en/translation.json:3156]:
```suggestion
    "tooltip": "Access Type of the client, for which the condition will be applied. Confidential client has enabled client authentication when public client has disabled client authentication. Bearer-only is a deprecated client type."
```

**Review by andymunro — CHANGES_REQUESTED** [review]
Thanks, @mposolda.  Just a few changes to suggest.

**Comment by mposolda** [comment]
@andymunro Applied all your suggestions. Thanks

**Review by andymunro — APPROVED** [review]

**Pull request merged**